common/autoware_ad_api_msgs/routing/srv/SetRoutePoints.srv

```diff
@@ -0,0 +1,9 @@
+std_msgs/Header header
+geometry_msgs/Pose[<=1] start
```

Comment:
Same here. When do people specify the `start`?

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/1494#discussion_r940867533

Reply:
Removed. https://github.com/autowarefoundation/autoware.universe/pull/1494#discussion_r942091088